src/sharepoint/account/add/sharepoint-account-add.html

```diff
@@ -1,51 +1,34 @@
-                        'has-success': addAccountCtrl.addAccountForm.$valid && addAccountCtrl.addAccountForm.$dirty}">
-                    <div class="clearfix">
-                        <label class="control-label col-md-4" for="optionPricesQuantity-{{$index}}"
-                               data-translate="{{ 'sharepoint_accounts_action_sharepoint_add_product_' + option.planCode }}"></label>
-                        <div class="col-md-3">
-                            <div data-wuc-increment-number
-                                 data-ng-model="option.prices[0].quantity"
-                                 data-ng-required="true"
-                                 data-wuc-increment-number-id="optionPricesQuantity-{{$index}}"
-                                 data-wuc-increment-number-init="option.prices[0].minimumQuantity"
-                                 data-wuc-increment-number-min="option.prices[0].minimumQuantity"
-                                 data-wuc-increment-number-max="option.prices[0].maximumQuantity"
-                                 data-wuc-increment-number-name="optionPricesQuantity[{{$index}}]"
-                                 data-wuc-increment-number-size="2">
-                            </div>
-                        </div>
-                        <div class="col-md-4 col-md-offset-1 mt-2">
-                            <strong class="text-danger"
-                                    data-ng-bind-html="addAccountCtrl.constructor.getPrice(option) | currency : addAccountCtrl.constructor.getCurrency(option)"></strong>
-                            <span data-ng-bind-html="'sharepoint_accounts_action_sharepoint_add_monthly' | translate"></span>
-                        </div>
-                    </div>
-                    <small class="col-md-offset-4 pl-4 help-block"
-                           data-translate="sharepoint_accounts_action_sharepoint_add_error_outofbounds"
-                           data-translate-values="{ t0: option.prices[0].minimumQuantity, t1: option.prices[0].maximumQuantity }"
-                           data-ng-if="addAccountCtrl.addAccountForm.$invalid && addAccountCtrl.addAccountForm.$dirty"></small>
+<div data-ng-controller="SharepointAccountAddCtrl as $ctrl">
+    <oui-modal data-heading="{{ 'sharepoint_accounts_action_sharepoint_add_title' | translate }}"
+               data-primary-action="$ctrl.submit()"
+               data-primary-label="{{ 'wizard_confirm' | translate }}"
+               data-secondary-action="resetAction()"
+               data-secondary-label="{{ 'common_cancel' | translate }}"
+               data-on-dismiss="resetAction()"
+               data-loading="$ctrl.loading">
+        <p data-translate="sharepoint_accounts_action_sharepoint_add_intro"></p>
+        <form class="form-horizontal" name="$ctrl.addAccountForm" novalidate
+              data-ng-if="!$ctrl.loading && $ctrl.price">
+            <div class="clearfix">
```

Comment:
bootlint: E014  Columns (`.col-*-*`) can only be children of `.row`s or `.form-group`s 

Documentation: https://github.com/twbs/bootlint/wiki/E014

You can get rid of `<div class="clearfix">`.